common/src/main/resources/assets/viaversion/config.yml

```diff
@@ -170,6 +170,9 @@ hide-scoreboard-numbers: false
 # Fixes 1.21+ clients on 1.20.5 servers placing water/lava buckets at the wrong location when moving fast, NOTE: This may cause issues with anti-cheat plugins.
 fix-1_21-placement-rotation: true
 #
+# If enabled, cancel swing packets sent while having an inventory opened on 1.15.2 and below servers. This can cause false positives with anti-cheat plugins.
```

Comment:
Why not make this `false` if it causes anti-cheat concerns?

Reply:
Because it also fixes things / don't want to change the default value / behaviour.